docs/pages/blog/2020-q3-update.md

```diff
@@ -151,7 +151,7 @@ We'll do our best, no guarantee!
 
 ### Community
 
-- 🗓 Execute on all the items of the [public roadmap](https://github.com/orgs/mui/projects/18/views/1).
+- 🗓 Execute on all the items of the [public roadmap](https://github.com/orgs/mui/projects/23/views/12).
```

Comment:
I'm just linking to the Material UI board for some of these older posts, as we were primarily referring to it years ago.